kernel/src/process_checker/basic.rs

```diff
@@ -83,6 +83,7 @@ impl<'a> AppCredentialsPolicy<'a> for AppCheckerSimulated<'a> {
     }
 }
 
+#[allow(dead_code)]
```

Comment:
?

Reply:
It's a warning we get on newer compilers that for some reason maybe we weren't getting before? Either way, the code is never used and should probably actually be removed, but I didn't want to mess with that stuff in this PR. (Document in the changing commit, but I know that's somewhat hidden)

Reply:
I think it should be `pub`.